Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.0.0'
+ruby '2.4.7'
```

Comment:
@sethkrasnianski 2.4.9 is available.  Anything preventing us from going up to that patch level?

Reply:
Nope. Lemme update.

Reply:
@joealba updated.